PendingReleaseNotes

```diff
@@ -118,16 +116,16 @@
   recommend that users with versioned buckets, especially those that existed
   on prior releases, use these new tools to check whether their buckets are
   affected and to clean them up accordingly.
-* rgw: The User Accounts feature unlocks several new AWS-compatible IAM APIs
-  for the self-service management of users, keys, groups, roles, policy and
+* RGW: The user accounts feature unlocks several new AWS-compatible IAM APIs
```

Comment:
What do you think of this?  `The "user accounts" feature` ?

That keeps this complex adjective unambiguous in the mind of the first-time reader.

Let me know what you think.

Reply:
Points taken and revisions pushed